definitions/ext-db2/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: DB2
+synthesis:
+  name: database
```

Comment:
One doubt is if this should be instance.

Since other db's seem to use instance not database name.

Reply:
Hi @sri-shetty with the db2 definition the issue is what Ignacio mentioned here, could you share a sample of the telemetry for us to check it? is the 'database' attribute going to be unique between other instances of this type?

Reply:
@IreneMLC - database attribute in the db2 event type is unique. I have uploaded the sample events for db2 into staging. Here is the permalink to staging. [Staging db2 event type](https://staging-one.newrelic.com/-/0nVjYAKYlj0). I'll email you the json containing db2 events.

Reply:
Thanks for the information, since you have the domain knowledge we can proceed with this. If you detect any issues you can modify the attributes used to synthesize the entity, the only consequence of that is that the GUID will change if the identifier is changed.